src/test/java/es/deusto/spq/payment/PayPal/connections/ServerListenerTest.java

```diff
@@ -0,0 +1,41 @@
+package es.deusto.spq.payment.PayPal.connections;
+
+import java.io.IOException;
+
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import es.deusto.spq.payment.PayPal.PayPal;
+
+public class ServerListenerTest {
+
+	/** An instance of the class to be tested. */
+	private static ServerListener serverListener;
+	/** Instanced to load data. */
+	@SuppressWarnings("unused")
+	private static PayPal paypal;
+	
+	/**
+	 * Initializes the necessary data to execute the test.
+	 * @throws IOException - launched by {@code java.net}.
+	 */
+	@BeforeClass
+	public static void initialize() throws IOException {
+		paypal = new PayPal();
+		serverListener = new ServerListener(45000);
```

Comment:
Tests seem to be conflicting with the use of the same port. What about changing it for this one to, say, 45001?
That way it doesn't fail in my end.